src/domains/Townhall/TownhallSettings/RatingSettings.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Switch } from '@material-ui/core';
+
+import SettingsList from 'components/SettingsList';
+import SettingsItem from 'components/SettingsItem';
+
+import text from './help-text';
+import { Props, areEqual } from './utils';
+
+export default React.memo(function ChatSettings({
+    onChange,
+    value,
+}: Props<'rating'>) {
+    const ref = React.useRef(value);
```

Comment:
Could you change this variable name to something more descriptive?  (I'm guilty of doing this in other code so I feel bad asking this)